packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.mdx

```diff
@@ -0,0 +1,35 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as viewerStories from './rich-text-viewer.stories';
+import * as mentionUserStories from '../../../rich-text-mention/users/tests/rich-text-mention-users.stories';
+import * as userMappingStories from '../../../mapping/user/tests/mapping-user.stories';
+
+<Meta of={viewerStories} />
+
+<Title of={viewerStories} />
+<Description of={viewerStories} />
+
+<DocsStory of={viewerStories.richTextViewer} expanded={false} />
+<Controls of={viewerStories.richTextViewer} />
+
+# Usage Docs
+
+## User Mention Usage
+
+To enable support for `@mention` in the `nimble-rich-text-viewer` component, the following configuration and mapping components should be added as children to convert the input markdown into a mention node, rendering a user name. Also, see [Client Usage Guidance on Filtered Users](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/rich-text/specs/mention-hld.md#client-usage-guidance-on-filtered-users-1) for instructions on loading the mapping elements into the viewer component. Follow the document for specific APIs in the below configuration and mapping elements.
+
+## User Configuration
+
+<Description of={mentionUserStories} />
```

Comment:
This description includes some text that's specific to the editor rather than the viewer (talks about the @mention button and keystroke). We should either make the shared text more general or export separate descriptions for the editor and the viewer.

Reply:
Updated the description to be more generic in the user mention storybook. Please let me know if there are any corrections in the doc.